src/coordinated_workers/coordinator.py

```diff
@@ -132,23 +132,21 @@ class ClusterRolesConfig:
 
     def __post_init__(self):
         """Ensure the various role specifications are consistent with one another."""
-        are_meta_keys_valid = set(self.meta_roles.keys()).issubset(self.roles)
-        are_meta_values_valid = all(
-            set(meta_value).issubset(self.roles) for meta_value in self.meta_roles.values()
-        )
-        is_minimal_valid = set(self.minimal_deployment).issubset(self.roles)
-        is_recommended_valid = set(self.recommended_deployment).issubset(self.roles)
+        error_message = None
```

Comment:
I think I get why you're doing this, but I wish this was done in a different PR. adding drive-bys to a 50-file-diff PR doesn't make it easier to review :)